pages/about/Calendar.html

```diff
@@ -25,6 +25,7 @@
             <div class="ui padded basic segment">
             <h1 class="ui block header center blue aligned">School Calendar</h1>
             <div class="ui blue center aligned segment">
+            <div class="ui blue segment">
```

Comment:
@denispop9 please remove this line as it is not necessary, ping me when you're done.